tests/unit/test_lidar_parser.py

```diff
@@ -102,6 +99,8 @@ def test_oscillation_detected_up_to_down(
         result, oscillation = lidar_parser_instance.run(lidar_detection_down)
         assert result
         assert oscillation is not None
+        assert len(oscillation.readings) == 2
+        assert lidar_parser_instance.lidar_readings[0].angle == -15.0
```

Comment:
check that the oscillation has this angle in it, not the instance that you created